lib/Payplug/Authentication.php

```diff
@@ -144,4 +174,98 @@ private static function validateToken(Payplug $payplug)
             throw new ConfigurationException('The Payplug configuration requires a valid token.');
         }
     }
+
+    /**
+     * Retrieve client datas from the user manager resource.
+     *
+     * @param   Payplug $payplug the client configuration
+     *
+     * @return  array the client id and client_secret_mask
+     *
+     * @throws  Exception
+     */
+    public static function getClientData($session = null, Payplug $payplug = null)
```

Comment:
Warning... please test this carefully that is not being used in any case for the moment for real merchants

Reply:
Thank you @dmurillo-payplug  for this warning. This method is not used plugin side getClientData we keep it just in case we need  to retrieve those information in the future